app/services/fast_track/hypertension_pdf_generator.rb

```diff
@@ -2,31 +2,41 @@
 
 module FastTrack
   class HypertensionPdfGenerator
-    attr_accessor :patient, :bp_data, :medications
```

Comment:
I looked around and can't find any reason to make these accessible outside the class. I think until we need it we should hold off from making them accessible.